### PR TITLE
fix tracker erratum linking

### PR DESCRIPTION
**Pull request by osoukup**
This PR primarily fixes the erratum-tracker linking and brings a few more minor changes.

Closes OSIDB-2752

### CODE REVIEW

File: collectors/errata/core.py [hunk 1]
```diff
@@ -49,13 +51,20 @@ def get_erratum(et_id) -> dict:
...
         "advisory_name": erratum_json["advisory_name"],
         "created_dt": erratum_json["timestamps"]["created_at"],
         "shipped_dt": erratum_json["timestamps"]["actual_ship_date"],
         "updated_dt": erratum_json["timestamps"]["updated_at"],
     }
+    logger.info(f"Syncing erratum {erratum['advisory_name'] or erratum['et_id']}")
+    logger.debug(
+        f"Created: {erratum['created_dt']} "
+        f"Updated: {erratum['created_dt']} "
+        f"Shipped: {erratum['created_dt']}"
```
Comment (dmonzonis) [inline collectors/errata/core.py:65]: Why are they all showing `erratum['created_dt']`?

Comment (osoukup) [reply collectors/errata/core.py:65]: This is definitely a bug on my side :facepalm: thank you for spotting it!

File: collectors/errata/tests/test_core.py [hunk 2]
```diff
@@ -25,6 +32,43 @@ def test_get_all_errata(self):
...
+            (
+                timezone.datetime(2020, 1, 1, tzinfo=timezone.utc),
+                timezone.datetime(2021, 1, 1, tzinfo=timezone.utc),
+                timezone.datetime(2020, 1, 1, tzinfo=timezone.utc),
+            ),
+        ],
+    )
+    def test_get_batch_end(self, bz_updated, jira_updated, expected_end):
+        """
+        test that the batch end logic respect the dependenies
```
Comment (jobselko) [inline collectors/errata/tests/test_core.py:57]: typo `dependenies`

**Review by dmonzonis — APPROVED** [review]

File: collectors/errata/tests/test_core.py [hunk 3]
```diff
@@ -30,6 +32,43 @@ def test_get_all_errata(self):
 
         assert len(errata_id_name_pairs) == 3
 
+    @pytest.mark.parametrize(
+        "bz_updated,jira_updated,expected_end",
```
Comment (jobselko) [inline collectors/errata/tests/test_core.py:36]: `expected_dt` could not be parametrized as the date is same

**Review by jobselko — APPROVED** [review]

File: collectors/errata/tests/test_core.py (hunk 2 above)
Comment (osoukup) [reply collectors/errata/tests/test_core.py]: Thank you for spotting it! Fixing

**Pull request merged**